modules/custom/az_event/az_event.module

```diff
@@ -123,6 +123,24 @@ function az_event_node_view(array &$build, EntityInterface $entity, EntityViewDi
 
     // For the full content view mode, modify the display of az_event_date.
     if (($view_mode === 'full') && ($entity instanceof NodeInterface)) {
+      // Compute a summary for the description.
+      // Field_az_body does not have native support for summaries.
+      // Some integrations for adding calendar events depend upon the data
+      // being able to fit into an URL. (2048 characters)
+      // Prefer field_az_summary where available.
+      $summary = '';
+      if ($entity->hasField('field_az_summary')) {
+        /** @var \Drupal\Core\Field\FieldItemList $summary_field */
+        $summary_field = $entity->get('field_az_summary');
+        $summary = $summary_field->value ?: '';
```

Comment:
Would it be better to use the NULL coalesce operator here (instead of the ternary operator)?   (Same question for line 140)
```suggestion
        $summary = $summary_field->value ?: '';
```